test/parallel/test-dgram-send-callback-buffer.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+var common = require('../common');
+var assert = require('assert');
+
+var fs = require('fs');
+var dgram = require('dgram');
+var callbacks = 0;
+var client, timer, buf;
+
+client = dgram.createSocket('udp4');
+
+buf = new Buffer(256);
+
+client.send(buf, common.PORT, '127.0.0.1', function(err, bytes) {
```

Comment:
Use `common.mustCall` here, so that it will make sure that the test will fail if the callback is not invoked.